modules/virt-creating-localnet-nad-cli.adoc

```diff
@@ -61,15 +61,17 @@ spec:
             "type": "ovn-k8s-cni-overlay", <3>
             "topology": "localnet", <4>
             "mtu": 1500, <5>
-            "netAttachDefName": "default/localnet-network" <6>
+            "vlanID": 200, <6>
+            "netAttachDefName": "default/localnet-network" <7>
     }
 ----
 <1> The CNI specification version. The required value is `0.3.1`.
 <2> The name of the network. This attribute must match the value of the `spec.desiredState.ovn.bridge-mappings.localnet` field of the `NodeNetworkConfigurationPolicy` object that defines the OVS bridge mapping.
 <3> The name of the CNI plug-in to be configured. The required value is `ovn-k8s-cni-overlay`.
 <4> The topological configuration for the network. The required value is `localnet`.
 <5> Optional: The maximum transmission unit (MTU) value. If you do not set a value, the Cluster Network Operator (CNO) sets a default MTU value by calculating the difference among the underlay MTU of the primary network interface, the overlay MTU of the pod network, and byte capacity of any enabled features, such as IPsec.
-<6> The value of the `namespace` and `name` fields in the `metadata` stanza of the `NetworkAttachmentDefinition` object.
+<6> Optionally, for more fine-grained network management, you can configure a virtual LAN (VLAN) ID for the NAD. Pods that use this NAD will have have an interface whose traffic will be able to communicate only with other devices that use the same VLAN ID (`200` in this example).
```

Comment:
"_Pods that use this NAD will have have an interface..._"
1. My English grammar expertise is limited, so I might be wrong here, but I think that "_have have_" is wrong (should be "_have to have_"?).
2. This is a bit confusing. Although this sentence is true, the context of this doc is VMs, not pods. Maybe it should say something like "_Pods/VMs that use this NAD..._".

Reply:
@yossisegev , no, you're absolutely right, "have have" is a typo here. I will also rewrite the second sentence to speak about VMs.

Thank you for the quick review!